src/twisted/internet/defer.py

```diff
@@ -160,6 +166,65 @@ def maybeDeferred(f, *args, **kw):
 
 
 
+def unbox(deferred, raises=True, passthrough=True):
+    """
+    Unbox a L{Deferred} that has already fired its result.
+
+    Take a deferred that has already fired its result and unbox it, returning
```

Comment:
This docstring should also note what happens to the callback chain.  The result is *consumed*, whether it's a `Failure` or a success result.